apps/graphql/src/apps/itinerary/Itinerary.js

```diff
@@ -36,6 +36,32 @@ export type ItinerariesOneWaySearchParameters = {|
     +infants?: number,
   |},
 |};
+
+export type ItinerariesReturnSearchParameters = {|
```

Comment:
There are a lot of common types with the `ItinerariesOneWaySearchParameters`,
what about creating types for them? 

For example 
```
type Sort = 'price' | 'duration' | 'quality' | 'date' | 'popularity',
type Order = 'ASC' | 'DESC',
type DateRange = {
 +start: Date,
 +end?: Date,
}
type Passengers: {...}
```

Reply:
Done!

Reply:
Thanks